src/kbc_modules/cc_kbc/mod.rs

```diff
@@ -215,23 +220,25 @@ impl Kbc {
         bail!("Request KBS resource: Attested but KBS still return Unauthorized")
     }
 
-    /// Convert a [`ResourceUri`] to the address of kind cc-kbc.
-    /// This function will **ignore** the kbs address the kid carries,
-    /// instead overwrite with the kbs_uri the [`Kbc`] carries.
-    /// Related issue: <https://github.com/confidential-containers/attestation-agent/issues/130>
-    pub fn resource_web_addr(&self, kid: &ResourceUri) -> Result<String> {
-        if self.kbs_uri != kid.kbs_addr {
-            warn!(
-                "The KBS address contained in the URI is not the same as the KBC's, use the KBC's.",
-            )
+    /// Convert a [`ResourceUri`] to a KBS URL.
+    pub fn resource_to_kbs_uri(&self, resource: &ResourceUri) -> Result<String> {
```

Comment:
do we want to have a unit test for the conversion?

Reply:
We do. I'll add one.

Reply:
Done now.